styles.css

```diff
@@ -271,6 +271,16 @@ hr {
     border: solid 3px gold;
     cursor: grab;
 }
+button:disabled {
+    opacity: 20%;
+    fill: gray;
+}
+button:disabled:hover {
+    cursor: not-allowed!important;
+    opacity: 20%!important;
+    fill: gray!important;
+    border: solid 3px black!important;
+}
```

Comment:
Are the `!important` keywords necessary here? 

You could try switching the position of this css rule with `button:disabled` and see if you can take them out. 

If you need to leave them in, toss a space between the value and the `!important` keyword

Example
`cursor: not-allowed !important`

Reply:
Word. Sounds good. It's not a huge deal, so don't spend too much time on it. Keeping them is fine.

Reply:
this one isn't resolved yet